src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -120,6 +120,14 @@ public class HubEndPoint<THub> : IInvocationBinder where THub : Hub
                             ? (IDataEncoder)Base64Encoder
                             : PassThroughEncoder;
 
+                        var transferMode = connection.Features.Get<ITransferModeFeature>() ??
+                            throw new InvalidOperationException("Unable to read transfer mode.");
+
+                        transferMode.TransferMode =
```

Comment:
Where is this used? Or are we just updating it to be correct? I can't see who uses this thing.